### PR TITLE
Update the German translations for GSA 7.0

### PR DESCRIPTION
**Pull request by timopollmeier**
The labels in the "Duration Timeline" chart and a few other elements were lacking German translations,
 which should be fixed with these changes.

### CODE REVIEW

File: src/html/classic/js/gsa_chart_helpers.js [hunk 1]
```diff
@@ -1587,25 +1591,37 @@
...
+          = label + gch.field_name(info.column ? info.column : info.stat,
+                                   info.type);
+
     if (include_stat) {
       switch (info.stat) {
         case 'min':
-          label = label + 'min. ';
+          label = label + gsa._('min. {{field_name}}',
+                                {field_name: field_name,
+                                 interpolation: {escape: false}});
```
Comment (bjoernricks) [inline src/html/classic/js/gsa_chart_helpers.js]: After looking at https://www.i18next.com/interpolation.html did they changed `escape` to `escapeValue`? I have always used `_('min. {{- field_name}}')` until now. Did you check if this works?

Comment (bjoernricks) [reply src/html/classic/js/gsa_chart_helpers.js]: Just saw `escape` expects a function https://www.i18next.com/interpolation.html#all-interpolation-options. Therefore your changes shouldn't work atm.

**Comment by timopollmeier** [comment]
I didn't see any problems with the current use cases but I've changed it to use the "-" prefix
 for unescaped variables.

**Comment by bjoernricks** [comment]
Thanks. Next time it would be nice to update your pull request by changing the history with git rebase -i instead of adding additional fix commits.

**Pull request merged**